File: Backend/Dot Net/LoginWebRestApi/LoginWebRestApi/Models/Area.cs
using System;
using System.Collections.Generic;

namespace LoginWebRestApi.Models;

public partial class Area
{
    public int Aid { get; set; }

    public string? Name { get; set; }

    public string? Pincode { get; set; }

    public int? Cityid { get; set; }

    public virtual City? City { get; set; }

    public virtual ICollection<Patient> Patients { get; set; } = new List<Patient>();
}

File: Backend/SpringBoot/HospitalManagementApi/src/main/java/com/example/demo/entity/Area.java
package com.example.demo.entity;




import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "area")
public class Area {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int aid;

    private String name;
    private String pincode;

    @ManyToOne
    @JoinColumn(name = "cityid", referencedColumnName = "cid")
    @JsonIgnoreProperties("areas")
    private City city;

   
}
